core/client/init_node.go

```diff
@@ -88,7 +88,7 @@ func (n *Node) ShouldUpdateNetwork() (bool, *conf.Network, error) {
 	if err != nil {
 		return false, nil, err
 	}
-	network, err := GetNetwork(n.clientCtx)
+	network, err := GetNetwork(n.clientCtx, cfg.BlockWorker)
```

Comment:
Why do we need this change?

Reply:
Because we need to pass blockworker url while initializing a network to check if it exists